contrib/IECoreArnold/include/IECoreArnold/CameraAlgo.h

```diff
@@ -45,7 +45,7 @@ namespace IECoreArnold
 namespace CameraAlgo
 {
 
-AtNode *convert( const IECore::Camera *camera );
+AtNode *convert( const IECore::Camera *camera, const std::string &nodeName, const AtNode *parentNode = nullptr );
```

Comment:
Do you think there's any merit in the node name being passed as `AtString`, as it is in the underlying `AiNode` calls? On the one hand it'd avoid an AtString conversion in the event that you had one handy already, but on the other hand, I can't think of any times we do have one already...

Reply:
Hmm, I don't think there's any good reason here ( node names are pretty user defined, so it's unlikely that we would be able to reuse them all that effectively ).

Now, where it would rather make sense is in the name parameter taken by convertP and convertPrimitiveVariable functions in ShapeAlgo.  There are definitely cases where prim var names are known in advance ( like where we sometimes hardcode that "P" is called "P" ), and I originally wrote this using AtString.  When I went to look at it afterwards, I thought you would probably consider this a public API that I shouldn't be changing, and it didn't seem worth fighting about, so I switched it to an internal cast to AtString.  But if you're in favour of using AtString in our function signatures where appropriate, that's the that I should switch back.

Reply:
It is public API, but now's the time to change it if we want to. I'd be happy with AtString for those if you think it's worth it...